lib/adhearsion-ivr/ivr_controller.rb

```diff
@@ -178,6 +192,20 @@ def prompts
       self.class.prompts
     end
 
+    def barge(val = nil)
+      if val.nil?
+        if @barge.nil?
+          self.class.barge
+        else
+          old = @barge
+          @barge = nil
```

Comment:
Why not move this to a separate method, something like `reset_barge`, even if it's private?